Vagrant/bootstrap.sh

```diff
@@ -92,3 +92,38 @@ $RUN_AS_INSTALLUSER git remote add origin "https://github.com/$gitrepo.git"
 $RUN_AS_INSTALLUSER git fetch --all
 $RUN_AS_INSTALLUSER git reset --hard origin/master
 $RUN_AS_INSTALLUSER bundle install
+
+# Setup the application
+
+# 1. Create a migration: rails generate migration CreateDoiRequests
+$RUN_AS_INSTALLUSER bundle exec rails generate migration CreateDoiRequests
+DOI_MIGRATION_FILE=`find db/migrate -type f -name '*_create_doi_requests.rb'|sort|tail -1`
+# 2. Replace the contents of the new migration with this gist: https://gist.github.com/tingtingjh/ab35348f493d565cdcc8
+$RUN_AS_INSTALLUSER cat > $DOI_MIGRATION_FILE <<GIST
+class CreateDoiRequests < ActiveRecord::Migration
+  def change
+    create_table :doi_requests do |t|
+      t.string "collection_id"
+      t.string "ezid_doi", default: "doi:pending", null: false
+      t.string "asset_type", default: "Collection", null: false
+      t.boolean "completed", default: false
+
+      t.timestamps null: false
+    end
+    add_index :doi_requests, :ezid_doi
+    add_index :doi_requests, :collection_id
+  end
+end
+GIST
+# 3. Generate Role model: rails generate roles
+$RUN_AS_INSTALLUSER bundle exec rails generate roles
+# 4. Remove the before filter added to app/controllers/application_controller.rb
+$RUN_AS_INSTALLUSER sed -i '/^  before_filter do$/,/^  end$/d' app/controllers/application_controller.rb
+# 5. Migrate
+$RUN_AS_INSTALLUSER bundle exec rake db:migrate
+# 6. Create default roles and an admin user
+$RUN_AS_INSTALLUSER bundle exec rake datarepo:setup_defaults
+# 7. Install Orcid
+$RUN_AS_INSTALLUSER bundle exec rails generate orcid:install --skip-application-yml
+# 8. Revert changes already incorporated
+$RUN_AS_INSTALLUSER git checkout ./app/models/user.rb ./config/routes.rb
```

Comment:
This should probably be the first step in another bootstrap script, since the orcid install will probably stop this one from completing.